doc-content/enterprise-only/installation/securing-passwords-was-proc.adoc

```diff
@@ -16,25 +16,12 @@ If {KIE_SERVER} is not configured with JCEKS, {KIE_SERVER} stores passwords in c
 * {KIE_SERVER} is installed in {WEBSPHERE}.
 * A {KIE_SERVER} user with the `kie-server` role has been created, as described in xref:kie-server-was-goup-proc[].
 * Java 8 or higher is installed.
+* An initialized JCEKS keystore is available.
 
 .Procedure
-. To use KeyTool to create a JCEKS, enter the following command in the Java 8 home directory:
-+
-[source,bash]
-----
-$<JAVA_HOME>/bin/keytool -importpassword -keystore <KEYSTORE_PATH> -keypass <ALIAS_KEY_PASSWORD> -alias <PASSWORD_ALIAS> -storepass <KEYSTORE_PASSWORD> -storetype JCEKS
-----
-+
-In this example, replace the following variables:
-
-* `<KEYSTORE_PATH>`: The path where the keystore will be stored
-* `<KEYSTORE_PASSWORD>`: The keystore password
-* `<ALIAS_KEY_PASSWORD>`: The password used to access values stored with the alias
-* `<PASSWORD_ALIAS>`: The alias of the entry to the process
-
+. Start the JCEKS keystore.
```

Comment:
I wouldn't mention this here, I would expect initialized JCEKS keystore to already contain stored password.

Reply:
Thanks. I've updated. Does it make sense to say 'Start the JCEKS keystore.? So that the next step makes sense?